VAMobile/src/api/personalInformation/getPersonalInformation.tsx

```diff
@@ -42,5 +43,6 @@ export const usePersonalInformation = (options?: { enabled?: boolean }) => {
     meta: {
       errorName: 'getPersonalInformation: Service error',
     },
+    staleTime: ACTIVITY_STALE_TIME,
```

Comment:
I set the stale time to this (5 minutes) to prevent the `v2/user` endpoint from being hit twice (once from the `NotificationManager` after the user is signed in, and another time from the `Nametag` component on the home screen when the about you section finishes loading). Since the default stale time for all queries is just 5 seconds, there was enough time in between for the personal info query to become stale before the `Nametag` was rendered.

The final stale time for this will be decided in #8638